src/Projectionist.php

```diff
@@ -248,6 +244,7 @@ public function isReplaying(): bool
     }
 
     public function replay(
+        string $storedEventClass,
```

Comment:
I feel like this parameter is not important enough to be put first. Instead, put it last.